packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -112,10 +110,12 @@ const chatMessageStyles: ComponentSlotStylesInput<
   content: ({ props: p, variables: v }): ICSSInJSStyle => ({
     color: v.contentColor,
     display: 'block',
-    '& a:focus': {
+    '& a': {
       outline: 'none',
-      color: v.contentFocusOutlineColor,
-      textDecoration: 'underline',
+      color: v.contentColorFocus,
```

Comment:
Can we rename this to something more meaningful to the usage, Like `linkColor`? This is not used in a context of focus, and it's not for all content, only the links..

Reply:
done, thanks